### PR TITLE
add skomer (Samsung XCover 2)

### PR DESCRIPTION
**Pull request by wimpknocker**

### CODE REVIEW

File: device-samsung-skomer-configs/var/lib/environment/compositor/droid-hal-device.conf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Config for samsung/skomer
+HYBRIS_EGLPLATFORM=fbdev
```
Comment (sledges) [inline device-samsung-skomer-configs/var/lib/environment/compositor/droid-hal-device.conf]: There is a bug in HADK PDF, please change to EGL_PLATFORM

File: droid-hal-device.inc [hunk 2]
```diff
@@ -603,6 +603,8 @@ fi
 # Droid config droppings
 /*.rc
 /default.prop
+/selinux_version
+/selinux_contexts
```
Comment (sledges) [inline droid-hal-device.inc]: These will break other devices who don't have these files. Please do this manually for now.
HADK is moving to modular layout where such things are resolved (PDF will be updated soon).
You need to migrate to modular at some point (hybris-11.0 is the last version to support non-modular layout), it's not difficult: http://piratepad.net/dhd2modular

File: extract-headers.sh [hunk 3]
```diff
@@ -207,6 +207,15 @@ check_header_exists bionic/libc/kernel/common/linux/sync.h && \
         bionic/libc/kernel/common/linux/sync.h \
         bionic/libc/kernel/common/linux/sw_sync.h
 
+check_header_exists bionic/libc/kernel/uapi/linux/sync.h && \
```
Comment (sledges) [inline extract-headers.sh]: This change is already in the modular layout: https://github.com/mer-hybris/droid-hal-device/commit/e0d7a4bfc9dc2a038103f727c20efcbb6f9761e1
So really advisable to migrate ;)

**Comment by sledges** [comment]
Please look at other commits in this repository, they are in the shape of "[tag][optional-subtag] Message", so your commit could read "[skomer] Add Samsung XCover 2"

**Comment by wimpknocker** [comment]
hey, PR is now corrected :)

**Comment by sledges** [comment]
thanks merging! Check out the new hadk, and also https://github.com/mer-hybris/dhd2modular to migrate this packaging of yours to a new layout

**Pull request merged**